### DISCUSSION TITLE
is there anyway to support cocos 2.x version

### DISCUSSION THREAD

**Discussion by wanghaisheng** [ton-org/game-engines-sdk#12] [Q&A] [answered]
i find the demo is 3.8.3

**Comment by barinbritva** [comment, marked as answer]
@wanghaisheng hi,

Unfortunately, the lib can be used only with 3.8.3 for now. As I know Cocos team working on wider support.
I will let you know in this Discussion if there is any updates

**Reply by wanghaisheng** [reply]
》 As I know Cocos team working on wider support.
is this discussion in their forum?

**Reply by barinbritva** [reply]
I'm not sure if there is the topic on Cocos forum. We discussed it in chat

**Reply by barinbritva** [reply]
Just in case, here is a related issue - https://github.com/CocosTechLabs/flappy-bird/issues/4
